src/app/components/formElements/Slider/Slider.tsx

```diff
@@ -0,0 +1,42 @@
+'use client';
+
+import {
+  Controller,
+  FieldValues,
+  Path,
+  PathValue,
+  useFormContext,
+} from 'react-hook-form';
+import { SliderProps } from '@/app/interfaces/FormElements';
+import SliderInput from './SliderInput';
+
+const Slider = <TFieldValues extends FieldValues>(
+  rootProps: SliderProps<TFieldValues>
+) => {
+  const { name, defaultValue, disabled, rules: _rules } = rootProps;
+  const methods = useFormContext<TFieldValues>();
+  const rules = { ..._rules };
+
+  return (
+    <Controller
+      control={methods.control}
+      name={name}
+      defaultValue={defaultValue as PathValue<TFieldValues, Path<TFieldValues>>}
```

Comment:
When we receive `undefined` `defaultValue`, it triggers an error. Load the `/components` page and check the console.

Resort to default value at the very beginning in that case.